dm/tests/openapi/run.sh

```diff
@@ -210,23 +211,42 @@ function test_dump_task() {
 	# create dump task success
 	openapi_task_check "create_dump_task_success"
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
-		"query-status $task_name" \
+		"query-status $task_name_dump" \
 		"\"stage\": \"Stopped\"" 1
-	openapi_task_check "check_task_stage_success" $task_name 1 "Stopped"
 
 	init_dump_data
 
 	# start dump task success
-	openapi_task_check "start_task_success" $task_name ""
+	openapi_task_check "start_task_success" $task_name_dump ""
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status $task_name_dump" \
+		"\"stage\": \"Running\"" 1
```

Comment:
maybe cannot get the `Running` stage, since sometime it will finish immediately. I think we can remove check `Running`

Reply:
Also i have add a function to check stage by openapi. In this case using openapi check stage will be better